projects/lf-documentation/src/app/getting-started-cdn-example/getting-started-cdn-example.component.html

```diff
@@ -0,0 +1,154 @@
+})
+
+<span class="code-ts-import">export class</span> ExampleApp <span class="code-ts-import">implements</span> AfterViewInit &#123;
+
+<span class="code-ts-comment">// allows access to the LfChecklistElement in the html above, available starting in ngAfterViewInit hook</span>
+@ViewChild('myChecklist') <span class="code-ts-key-name">checklist!</span>: ElementRef
+
+<span class="code-ts-comment">// type Checklist is the type of the custom element imported from the NPM package</span>
+<span class="code-ts-key-name">currentChecklist: Checklist[];</span>
+
+<span class="code-ts-comment">// events for the elements are received as CustomEvents </span>
+<span class="code-ts-function">onChecklistChanged</span>(event: CustomEvent &lt;Checklist>	) &#123;
+  <span class="code-ts-import">const</span> checklist = event.<span class="code-ts-key-name">detail</span>;   <span class="code-ts-comment"> // data will be stored in the detail property</span>
+}
+
+<span class="code-ts-function">ngAfterViewInit()</span> &#123;
+<span class="code-ts-comment">// the element's view is initialized after ngAfterViewInit</span>
+this.<span class="code-ts-key-name">checklist.nativeElement</span>.<span class="code-ts-function">initAsync</span>(...)
+}
+}
+    </pre>
+  </li>
+</ol>
+
+<h3 id="cdn-example-react"> Example: <code>lf-checklist</code> In a React Project Via CDN</h3>
+<p>In the example below, the implementation comes from the CDN.
+<ol>
+  <li>
+    <p  class="documentation-text">
+      Add the following script tags to <code>./public/index.html</code>
+    </p>
+    <pre class="code-container">
+<span class="code-html-tagname"> &lt;script </span><span class="code-html-attribute-name">src</span><span>=</span><span class="code-html-attribute-value"
+>"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/laserfiche-ui-components.js"</span> <span class="code-html-attribute-name"
+>defer</span><span class="code-html-tagname">&gt;&lt;/script&gt;</span>
+<span class="code-html-tagname"> &lt;link </span><span class="code-html-attribute-name">href</span><span>=</span><span class="code-html-attribute-value"
+>"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/lf-laserfiche-lite.css"</span><span class="code-html-attribute-name"> rel</span><span
+>=</span><span class="code-html-attribute-value">"stylesheet" </span><span class="code-html-tagname">/&gt;</span>
+<span class="code-html-tagname"> &lt;link </span><span class="code-html-attribute-name">href</span><span>=</span><span class="code-html-attribute-value"
+  >"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/indigo-pink.css"</span><span class="code-html-attribute-name"> rel</span><span
+  >=</span><span class="code-html-attribute-value">"stylesheet" </span><span class="code-html-tagname">/&gt;</span>
+    </pre>
+  </li>
+  <li>
+    If you are using TypeScript, add a file <code>./src/definitions.d.ts</code>. This enables custom elements in your React application.
```

Comment:
This allows you to use the custom UI component in your application

OR

This allows you to use the custom HTML Element in your application